### ISSUE TITLE
Either from exception

### ISSUE THREAD

**Issue by rparree** [monet/monet.js#210]
Hi,

Is there a way of creating an Either from an expression that might yield an exception?  Or perhaps are there plans to add a `Try` (with a `toEither`)? If there's demand for that (either a method like below or a full `Try` type), i could look into that and send a pull request.

```
function fromTry(fn){
    try{
        return Either.right(fn())
    } catch(e) {
        return Either.left(e)
    }
}
```

**Comment by cwmyers** [comment]
Yeah that would be awesome.  I'm surprised I hadn't done that already.  Let's start with your suggestion and then see whether a first class `Try` type makes sense.

**Comment by rparree** [comment]
Ok i will do so this coming weekend.

**Comment by ulfryk** [comment]
Just to make things consistent with other stuff here, let's make it `Either.fromTry(…)` ;)

**Comment by ulfryk** [comment]
Also some work (alias `attempt`) is laready started https://github.com/monet/monet.js/pull/189

(but seems to be abandoned)

**Comment by ulfryk** [comment]
snapshot released as monet@0.9.0-422

**Comment by wookieb** [comment]
How about this? Can we merge it and release 0.9.1?

**Comment by ulfryk** [comment]
@wookieb it's merged wil be released with 0.9.1
